packages/flutter/lib/src/widgets/icon.dart

```diff
@@ -89,7 +89,10 @@ class Icon extends StatelessWidget {
   /// This would be read out in accessibility modes (e.g TalkBack/VoiceOver).
   /// This label does not show in the UI.
   ///
-  /// See [Semantics.label];
+  /// See also:
+  ///
+  ///  * [Semantics.label] which is set with [semanticLabel] in the underlying
+  ///  [Semantics] node.
```

Comment:
nit: indent this to align with [Semantics.label] in previous line.

Reply:
Maybe `[Semantics] widget` to be a little more specific?

Reply:
done